api/candlepin-api-spec.yaml

```diff
@@ -500,6 +636,39 @@ components:
             principalName:
               type: string
 
+    DistributorVersionDTO:
+      description: A DTO representation of the DistributorVersion
+      allOf:
+        - $ref: '#/components/schemas/TimestampedEntity'
+        - type: object
+          required:
+            - name
+          properties:
+            id:
+              type: string
+            name:
+              type: string
+              minLength: 1
+            displayName:
+              type: string
+            capabilities:
+              type: array
+              uniqueItems: true
+              items:
+                $ref: '#/definitions/DistributorVersionCapabilityDTO'
+
+    DistributorVersionCapabilityDTO:
```

Comment:
One more thing I forgot to mention. Now that our DTOs don't extend CandlepinDTO, which had a `@JsonFilter` annotation that ensures the dynamic property filter is used during their serialization, we need to indirectly add that annotation to all DTOs we port to openapi.

So both DistributorVersionDTO and DistributorVersionCapabilityDTO should have a mixin added in the `JsonProvider.addMixInAnnotationsForDTOs` method just like all other DTOs

Reply:
Done.